### PR TITLE
Send side with every command, rather than forward with length

### PR DESCRIPTION
**Pull request by qwandor**

### CODE REVIEW

File: hovercontrol/src/hoverkite.rs [hunk 1]
```diff
@@ -124,16 +122,21 @@ impl Hoverkite {
...
-                "No serial ports available. Can't send command to {:?}: {:?}",
-                side, command
-            ),
-        }
+        let side_command = SideCommand { side, command };
+        let port = match (side, self.left_port.as_mut(), self.right_port.as_mut()) {
+            (Side::Left, Some(port), _) => port,
+            (Side::Left, None, Some(port)) => port,
+            (Side::Right, _, Some(port)) => port,
+            (Side::Right, Some(port), None) => port,
```
Comment (alsuren) [inline hovercontrol/src/hoverkite.rs:130]: `panic!("the primary is currently assumed to be Right")` or something? (or just leave it like this until we work out a way to have the boards discover their role)

File: hoverkite-protocol/src/command.rs [hunk 2]
```diff
@@ -127,19 +127,37 @@ impl Command {
     }
 }
 
+/// A command for a particular side.
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub struct SecondaryCommand(pub Command);
+pub struct SideCommand {
```
Comment (alsuren) [inline hoverkite-protocol/src/command.rs:132]: DirectedCommand? :trollface:

**Review by alsuren — APPROVED** [review]
this feels a lot better. I'll have a stab at the other side during the week if you want.

File: hoverkite-protocol/src/command.rs (hunk 2 above)
Comment (qwandor) [reply hoverkite-protocol/src/command.rs]: Done.

File: hovercontrol/src/hoverkite.rs (hunk 1 above)
Comment (qwandor) [reply hovercontrol/src/hoverkite.rs:132]: I've left it for now.

**Pull request merged**